sdk/storage/storage-blob/src/sas/AccountSASPermissions.ts

```diff
@@ -129,7 +129,7 @@ export class AccountSASPermissions {
   public write: boolean = false;
 
   /**
-   * Permission to create blobs and files granted.
+   * Permission to delete blobs and files granted.
```

Comment:
I believe this is auto generated from swagger, @kazrael2119 Could you help update the swagger also?